charts/datadog-crds/templates/datadoghq.com_datadogagents_v1beta1.yaml

```diff
@@ -9397,6 +6558,10 @@ spec:
                   type: object
               type: object
           type: object
+      type: object
+  version: v2alpha1
```

Comment:
is this because of the removal of `name: v2alpha1`?

Reply:
In the 'before' version it's on the line 28, set to `version: v1alpha1`, this change happened when we stopped generating `v1alpha1` CRD here https://github.com/DataDog/datadog-operator/pull/1242.